internal/server/models/encryption_test.go

```diff
@@ -51,7 +50,9 @@ func TestEncryptedAtRest(t *testing.T) {
 
 	assert.Assert(t, "don't tell" != result)
 	assert.Assert(t, "" != result)
-	assert.Assert(t, is.Len(result, 88)) // encrypts to this many bytes
+
+	// TODO: why did this length change to 139/140 bytes?
+	// assert.Assert(t, is.Len(result, 88)) // encrypts to this many bytes
```

Comment:
TODO: look into this test failure and restore the assertion

Reply:
I think I understand this test failure now. Previously the `RootKeyID` stored in the `encryption_keys` table was `db_at_rest`. Now it's the full path to the file that holds the key. I believe that accounts for the extra 52 bytes and explains why sometimes it's 1 byte less (the random temp directory path has 1 less character).

I need to fix this to store just the root key ID.

Reply:
Looking at this more, I think this is only a bug in the test. In the production code path we were always storing the full path in this field because our defaults set a full path: https://github.com/infrahq/infra/blob/c5b9fa43d6d910b5d55a75cd5a74c5ffeb6a7faf/internal/cmd/server.go#L94

I'll remove this assertion since we check the round trip already, which is the important thing for this test.